tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -2934,7 +2941,7 @@ lldbg_emit_global_variable(LL_DebugInfo *db, SPTR sptr, ISZ_T off, int findex,
         llObjtodbgAddUnique(*listp, db->gbl_obj_exp_mdnode);
     }
     ll_add_global_debug(db->module, sptr, mdref);
-    if (gbl.rutype == RU_BDATA && sc == SC_CMBLK) {
+    if ((gbl.rutype == RU_SUBR || gbl.rutype == RU_BDATA) && sc == SC_CMBLK) {
```

Comment:
Is this also related to Module's file/lineno debug info?

Reply:
No. It's for capturing debug-info for modules defined in a separate translation unit. I've discussed/asked this for clarification here https://flang-compiler.slack.com/archives/C9CFEFP0T/p1598247677003800.